src/services/user-service.js

```diff
@@ -0,0 +1,34 @@
+export const createUser = (name, username, password, confirmPassword, email) =>
+    fetch(`http://localhost:8080/api/register/${username}`, {
+        // TODO: reconcile for where I deleted extra attributes from path
+        method: 'POST',
+        body: JSON.stringify(
+            {
+                name: name,
+                username: username,
+                password: password,
+                confirmPassword: confirmPassword,
+                email: email
+            }),
+        credentials: "include",
+        headers: {
+            'content-type': 'application/json'
+        }
+    })
+        .then(response => response.json())
+
+
+export const logIn = (username, password) =>
+    fetch(`http://localhost:8080/api/login/${username}/${password}`)
```

Comment:
change the fetch path to heroku server. try to move this to fetch method to a service